akka-actor/src/main/scala/akka/actor/Timers.scala

```diff
@@ -103,6 +113,15 @@ abstract class AbstractActorWithTimers extends AbstractActor with Timers {
    */
   def startSingleTimer(key: Any, msg: Any, timeout: FiniteDuration): Unit
 
+  /**
+   * Same as [[startSingleTimer]],
+   * but accepts Java [[java.time.Duration]] instead of Scala ones.
```

Comment:
yeah I have to agree with @hepin1989; we have to take the duplication here, since we want Java API to feel "first class" and not keep people to "oh actually you have to read the scala one anyway"

Reply:
Ok, i will change it